School/views.py

```diff
@@ -138,11 +138,12 @@ def teacher_detail_edit(request, teacher_id):
 def group_detail(request, group_id=1):
     group = Group.objects.get(id=group_id)
     student = Student.objects.filter(group_id=group_id)
```

Comment:
Filter returns multiple objects, but the variable name suggests  that there is only one.